modules/http2/h2_workers.c

```diff
@@ -40,7 +40,7 @@ struct h2_slot {
     apr_thread_t *thread;
     apr_thread_mutex_t *lock;
     apr_thread_cond_t *not_idle;
-    volatile apr_uint32_t timed_out;
+    apr_uint32_t timed_out;
```

Comment:
`volative` (or `/* volatile */`) might be documentary still

Reply:
ack

Reply:
added